client/utils/account-fees.tsx

```diff
@@ -86,17 +86,19 @@ export const getCurrentFee = (
 };
 
 export const formatMethodFeesTooltip = (
-	accountFees: FeeStructure | undefined
+	accountFees: FeeStructure
 ): JSX.Element => {
 	if ( ! accountFees ) return <></>;
+	// If there is a discount, use that instead of the base fee.
+	const currentFee = getCurrentFee( accountFees );
```

Comment:
For clarity, can we rename this function to `getCurrentBaseFee`, please? And correspondingly, the local variable to `currentBaseFee`.

Reply:
Sure, I have updated this and other references to `currentBaseFee`. That is indeed more descriptive. 

Commit - [1ec4acd](https://github.com/Automattic/woocommerce-payments/pull/5299/commits/1ec4acd531489ad3f58d1b99b79a875f498f4862)